formats/ome-tiff/code.txt

```diff
@@ -28,11 +28,21 @@ the following steps are required:
 
 #. Read in the 8-byte header.
 #. Determine if the file is a valid TIFF, and if so, whether its byte
-   order is big endian or little endian. The first pair of bytes must
-   equal "II" (little endian, "Intel") or "MM" (big endian, "Motorola").
-   The next pair must equal 42 with the proper endianness.
-#. Determine the byte offset into the file of the first IFD. This
-   information is stored in bytes 4-7, with the proper endianness.
+   order is big endian or little endian. Bytes 0-1 must equal "II"
+   (little endian, "Intel") or "MM" (big endian, "Motorola"), which is
+   0x4D4D in the correct endian format.
```

Comment:
I am a bit puzzled by this. We have "... X or Y, which is Z ...". I think the 0x4D4D is specifically about the MM, and not the II at all? It's not obviously not about both; I don't know if a punctuation tweak would help.